packages/frontend/src/hooks/useFetch.ts

```diff
@@ -223,9 +280,14 @@ export function useRpcFetch<
   Method extends MethodOfRpc,
   Args extends null | Parameters<Method>,
 >(method: Method, args: Args): null | Ret<Method>
+
 /**
  * A (possibly) more convenient version of {@link useFetch},
  * when dealing with `BackendRemote.rpc` calls.
+ *
+ * This hook provides better TypeScript support when working with
+ * the backend RPC methods, as it can infer the correct argument types
+ * and return types from the method signature.
```

Comment:
```suggestion
 * Namely, it allows you to pass `BackendRemote.rpc` methods directly
 * as the `method` arg, without having to worry about the `this` value.
 *
 * @example
 * useRpcFetch(BackendRemote.rpc.getMessage, [accId, msgId])
```

They all provide type support.